scripts/git-track/python.txt

```diff
@@ -0,0 +1,5 @@
+**/*.py
+**/*.md
+*.py
+*.md
+*.txt
```

Comment:
.gitignore
.openapi-generator/

and do we need tox.ini? is that autogenerated?

Reply:
yes there's a tox.mustache, let's add that too

Reply:
Done :p